packages/image_picker/image_picker/CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
+## 1.0.0
+
+* **BREAKING CHANGE**: Removes the deprecated `get*` methods. Clients who have
+  not already done so will need to migrate to the `pick*` versions that use
+  `XFile` rather than `PickedFile` for return values.
+    * As this is the only change, we encourage authors of published packages
+      that depend on `image_picker` to consider using a constraint of
+      `'>=0.8.9 <2.0.0'` rather than `^1.0.0` when updating dependencies, to
+      avoid conflicts with packages that have not yet updated.
+
 ## 0.8.9
 
-* Adds `getMedia` and `getMultipleMedia` methods.
+* Adds `pickMedia` and `pickMultipleMedia` methods.
```

Comment:
While I generally don't want us changing old changelogs, I noticed that we forgot to adjust the names for the app-facing package, making this wrong, and it's actively confusing when combined with the discussion of removing `get` in 1.0.0 so I thought it was worthwhile here.

Reply:
good catch